code/modules/vore/eating/belly_vr.dm

```diff
@@ -515,16 +537,8 @@
 	if(!(content in internal_contents))
 		return
 	internal_contents -= content
-	target.internal_contents += content
-	if(content in items_preserved)
-		items_preserved -= content
-		target.items_preserved += content
```

Comment:
Psst I un-nukified the loot gurgle code so you may wanna slip this part back in or things may get kinda borky heh.